fiaas_deploy_daemon/crd/types.py

```diff
@@ -56,6 +58,24 @@ class Meta:
     spec = Field(FiaasApplicationSpec)
 
 
+class FiaasApplicationStatusResult(Model):
+    result = RequiredField(six.text_type)
+    observedGeneration = Field(int, 0)
+    logs = ListField(six.text_type)
+
+
+class FiaasApplicationStatusInline(Model):
```

Comment:
This feels like a hack.

The proper way to do this is probably to implement support for working with subresources of any resource that has those in the k8s library.